libqtile/backend/base.py

```diff
@@ -204,6 +204,10 @@ def info(self) -> Dict[str, Any]:
 
 
 class Window(_Window, metaclass=ABCMeta):
+    """A regular Window belonging to a client."""
+    def __repr__(self):
+        return "Window(%r, %i)" % (self.name, self.wid)
```

Comment:
Might be easier to read this way?

```suggestion
        return "Window(name=%r, wid=%i)" % (self.name, self.wid)
```

Reply:
I like it. Updated.